lib/rubocop/cop/style/multiline_if_modifier.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for uses of if/unless guard clauses on multiple line statements
+      #
+      # @example
+      #
+      #   # bad
+      #   {
+      #     result: 'this should not happend'
+      #   } unless cond
+      #
+      #   # good
+      #   { result: 'ok' } if cond
+      class MultilineIfModifier < Cop
+        include IfNode
+        include StatementModifier
+        include AutocorrectAlignment
+
+        MSG = 'Favor a normal %s-expression over a modifier' \
```

Comment:
Elsewhere in the code base, we refer to `if-statement` and `unless-statement`, so we should probably use that wording here.